docs/fleet/provision.md

```diff
@@ -1,278 +1,210 @@
-  ```
+Regardless whether provisioning happens through the captive web portal or a mobile app, the process follows the same pattern:
 
-  This command uninstalls Viam Agent, `viam-server`, the machine configuration file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
+{{<video webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with viam-agent." poster="/platform/provisioning-demo.jpg" class="alignright" max-width="400px" style="margin-left: 2rem">}}
 
-  For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
+1. When a machine that has `viam-agent` installed and `agent-provisioning` configured comes online, `agent-provisioning` creates a WiFi hotspot.
 
-### View Viam Agent logs
+   - The [`agent-provisioning` configuration](#configuration) is at <file>/etc/viam-provisioning.json</file.
+   - The [`viam-agent` configuration](/configure/agent/#configuration), if provided, is at <file>/etc/viam.json</file> alongside the rest of the machine configuration, including which fragments to use (if any).
 
-The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
-You can find these messages on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
+1. The end user uses their mobile device or computer and connects to the WiFi hotspot.
 
-Viam Agent only sends messages when your machine is online and connected to the internet.
-If your machine is offline, log messages are queued, and are sent to the Viam app once your machine reconnects to the internet.
+   - By default, the hotspot network is named `viam-setup-HOSTNAME`, where `HOSTNAME` is replaced with the hostname of your machine.
+     The WiFi password for this hotspot network is `viamsetup` by default.
+     You can customize these values in the machine's or fragment's [`viam-agent` configuration](/configure/agent/#configuration).
 
-These log messages include when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warnings encountered during operation.
+1. If the end user has a provisioning mobile app, they should now open that app to complete setup.
+   Alternatively, you will be redirected to a sign-in page.
+   If you are using a laptop or are not redirected, try opening [http://viam.setup/](http://viam.setup/) in a browser.
 
-## Provision a new machine
+1. In the captive web portal or mobile app, the user will be prompted to provide the network information for the machine.
 
-With the Viam Agent installed, your machine will either connect to a local WiFi network or will create its own WiFi hotspot, depending on your configuration.
+   - If there is no machine configuration at <file>/etc/viam.json</file>, the provisioning app or captive portal will also require the user to paste a machine credentials configuration file.
```

Comment:
Only when using the web portal will the user needs to manually copy the machine credentials. The mobile app does this automatically.

(I think I pointed this out in my last review, but apparently all those comments are disconnected since the rebase...)

Reply:
sorry